src/openrct2/ride/Vehicle.cpp

```diff
@@ -7421,6 +7472,15 @@ void Vehicle::UpdateAdditionalAnimation()
                 }
             }
             break;
+        case VEHICLE_ENTRY_ANIMATION_ANIMAL_FLYING:
+            UpdateAnimationAnimalFlying();
+            // prevents underflow of animationState timer
+            targetFrame = abs(_vehicleVelocityF64E08) >> 24;
+            if (targetFrame > animationState)
```

Comment:
Change this to `animationState = std::max(animationState - targetFrame, 0);` that is how we normally show bounded mathematics. The comment is useless everyone can see it's stopping underflows. It should state why we are doing something like `// make animation speed affected by vehicle speed`.